qiita_db/user.py

```diff
@@ -232,10 +233,15 @@ def create(cls, email, password, info=None):
         # create user default sample holder
         sql = ("INSERT INTO qiita.analysis "
                "(email, name, description, dflt, analysis_status_id) "
-               "VALUES (%s, %s, %s, %s, 1)")
+               "VALUES (%s, %s, %s, %s, 1) RETURNING analysis_id")
         conn_handler.add_to_queue(queue, sql,
                                   (email, '%s-dflt' % email, 'dflt', True))
 
+        portal_id = convert_to_id('QIITA', 'portal_type', 'portal')
```

Comment:
That is something we need to discuss. The default to me is supposed to reside on qiita only, but do we then need defaults for each portal in order to track things correctly?